test_memusage is not buildable on non-linux systems.

will be submitted to upstream

--- openscap-1.3.7/tests/API/probes/CMakeLists.txt
+++ openscap-1.3.7/tests/API/probes/CMakeLists.txt
@@ -31,11 +31,13 @@ )
 target_link_libraries(oval_fts_list openscap)
 add_oscap_test("fts.sh")
 
-add_oscap_test_executable(test_memusage
-	"test_memusage.c"
-	"${CMAKE_SOURCE_DIR}/src/common/bfind.c"
-)
-target_include_directories(test_memusage PUBLIC
-	"${CMAKE_SOURCE_DIR}/src/common"
-)
-add_oscap_test("test_memusage.sh")
+if(OS_LINUX)
+	add_oscap_test_executable(test_memusage
+		"test_memusage.c"
+		"${CMAKE_SOURCE_DIR}/src/common/bfind.c"
+	)
+	target_include_directories(test_memusage PUBLIC
+		"${CMAKE_SOURCE_DIR}/src/common"
+	)
+	add_oscap_test("test_memusage.sh")
+endif(OS_LINUX)
